src/Helper/Input.php

```diff
@@ -23,12 +23,11 @@ class Input extends HelperLocator
      *
      * Given an input specification, returns the HTML for the input.
      *
-     * @param array $spec The element specification.
-     *
-     * @return mixed
+     * @param array{type?: string, name?: string, attribs?: array<string, scalar|null>, value?: mixed, options?: array<string, string>}|null $spec HTML input element specification
      *
+     * @return self|string
      */
-    public function __invoke(array $spec = null)
```

Comment:
Bit confused, can we do array $spec = [] to fix this ? I didnot noticed where the casting was done . (array) $attr mentioned. May be I missed.

Reply:
@harikt Gentle reminder

Reply:
My comment about doing [] is also incorrect as we change the signature and that will break